src/site/components/up_to_top_button.html

```diff
@@ -16,7 +16,7 @@
     <div class="usa-grid usa-grid-full">
         <div class="usa-width-three-fourths" id="usa-width-container">
             <div class="usa-content">
-                <button class="usa-button vads-u-background-color--gray-dark vsa-top-button vads-u-display--flex vads-u-align-items--center">
+                <button type="button" class="usa-button vads-u-background-color--gray-dark vsa-top-button vads-u-display--flex vads-u-align-items--center">
                     <i class="fas fa-arrow-up vads-u-margin-right--0 medium-screen:vads-u-margin-right--1p5 medium-screen:vads-u-font-size--sm"></i>
```

Comment:
For icons we need to add some attributes for accessibility, to hide it for screen reader users:

```diff
<i 
+  role="img" 
+  aria-hidden="true" 
class="fas fa-arrow-up vads-u-margin-right--0 medium-screen:vads-u-margin-right--1p5 medium-screen:vads-u-font-size--sm"></i>
```

Reply:
done